packages/odyssey-design-tokens/src/focus.json

```diff
@@ -0,0 +1,7 @@
+{
+  "focus": {
+    "outline": {
+      "width": { "value": "2px" }
```

Comment:
Would you mind bringing this reference into `Box`? I believe it's currently setting FocusOutlineWidth locally.